src/LaunchDarkly/LDClient.php

```diff
@@ -280,6 +280,16 @@ public function secureModeHash($user) {
     }
 
     /**
+     * Publish events to LaunchDarkly
+     * @param $payload string Event payload
```

Comment:
Was this parameter listed unintentionally? I don't see it specified in code.

Reply:
I tried something with a payload, and removed it, I guess I forgot the comment there. Submitted an update